cmd/meroxa/root/connectors/create.go

```diff
@@ -111,25 +111,31 @@ func (c *Create) CreateConnector(ctx context.Context) (*meroxa.Connector, error)
 		return nil, errors.New("requires either a source (--from) or a destination (--to)")
 	}
 
+	if c.flags.Pipeline == "" {
```

Comment:
Doesn't the `required: true` flag takes care of the pipeline flag existing?

Reply:
@dianadoherty unfortunately not in this scenario since this check is for the `connect` command :(

That command calls the `CreateConnector` function which goes separately to flags. In this scenario, `required: true` is only useful for the `connectors create` command.